.svglintrc.mjs

```diff
@@ -46,7 +47,7 @@ const sortObjectByKey = (obj) => {
 
 const sortObjectByValue = (obj) => {
   return Object.keys(obj)
-    .sort((a, b) => ('' + obj[a]).localeCompare(obj[b]))
+    .sort((a, b) => collator.compare(`${obj[a]}`, obj[b]))
```

Comment:
```suggestion
    .sort((a, b) => collator.compare(obj[a], obj[b]))
```

Reply:
It seems that is not needed, are always all strings.